src/common.h

```diff
@@ -5,6 +5,18 @@
 #include <config.h>
 #endif
 
+#ifdef has_jemalloc
```

Comment:
you probably want to define our own ENABLE_JEMALLOC in configure.ac to make sure conventions are set.